api/PayerErrors.json

```diff
@@ -112,9 +112,16 @@
           "safari_ios": {
             "version_added": null
           },
-          "samsunginternet_android": {
-            "version_added": "6.0"
-          },
+          "samsunginternet_android": [
+            {
+              "version_added": "10.0"
+            },
+            {
+              "alternative_name": "PayerErrorFields",
```

Comment:
This file needs research (or reverting) it seems to be overwriting the 6.0 version added which is probably correct?